pkg/stores/sqlproxy/proxy_store.go

```diff
@@ -87,6 +112,11 @@ var (
 		},
 		gvkKey("cluster.x-k8s.io", "v1beta1", "Machine"): {
 			{"spec", "clusterName"}},
+		gvkKey("management.cattle.io", "v3", "Cluster"): {
+			{"spec", "internal"},
+			{"spec", "displayName"},
+			{"status", "nodeName"},
```

Comment:
I don't think this one was requested in the issues and i couldn't find it on the resource itself (though only looking at local cluster in rancher docker install)

Reply:
indeed wrong copypaste, fixed as 3c0b461